plugins/linux/l3plugin/linuxcalls/arp_linuxcalls.go

```diff
@@ -33,60 +19,42 @@ package linuxcalls
 import (
 	"time"
 
-	"github.com/ligato/cn-infra/logging"
-	"github.com/ligato/cn-infra/logging/measure"
 	"github.com/vishvananda/netlink"
 )
 
 // AddArpEntry creates a new static ARP entry
-func AddArpEntry(name string, arpEntry *netlink.Neigh, log logging.Logger, timeLog measure.StopWatchEntry) error {
-	log.Debugf("Creating new ARP entry %v", name)
-	start := time.Now()
-	defer func() {
-		if timeLog != nil {
-			timeLog.LogTimeEntry(time.Since(start))
-		}
-	}()
+func (handler *netLinkHandler) AddArpEntry(name string, arpEntry *netlink.Neigh) error {
+	defer func(t time.Time) {
+		handler.stopwatch.TimeLog("add-arp-entry").LogTimeEntry(time.Since(t))
+	}(time.Now())
 
 	return netlink.NeighAdd(arpEntry)
 }
 
 // ModifyArpEntry updates existing arp entry
-func ModifyArpEntry(name string, arpEntry *netlink.Neigh, log logging.Logger, timeLog measure.StopWatchEntry) error {
-	log.Debugf("Deleting an ARP entry %v", name)
-	start := time.Now()
-	defer func() {
-		if timeLog != nil {
-			timeLog.LogTimeEntry(time.Since(start))
-		}
-	}()
+func (handler *netLinkHandler) ModifyArpEntry(name string, arpEntry *netlink.Neigh) error {
+	defer func(t time.Time) {
+		handler.stopwatch.TimeLog("modify-arp-entry").LogTimeEntry(time.Since(t))
+	}(time.Now())
 
 	return netlink.NeighSet(arpEntry)
 }
 
 // DeleteArpEntry removes an static ARP entry
-func DeleteArpEntry(name string, arpEntry *netlink.Neigh, log logging.Logger, timeLog measure.StopWatchEntry) error {
-	log.Debugf("Deleting an ARP entry %v", name)
-	start := time.Now()
-	defer func() {
-		if timeLog != nil {
-			timeLog.LogTimeEntry(time.Since(start))
-		}
-	}()
+func (handler *netLinkHandler) DeleteArpEntry(name string, arpEntry *netlink.Neigh) error {
```

Comment:
Let's keep just `DelArpEntry` like other deletion API, for example `DelInterfaceIP`.